config/arc_cluster/README.md

```diff
@@ -0,0 +1,37 @@
+# Running open-gira on ARC
+
+## Persistence
+
+Either create or attach to a tmux session:
+`tmux attach-session -t <id>`
+`tmux`
```

Comment:
Add a couple of notes on tmux

- `Ctrl+b` is used to start commands, often written as `C-b` in documentation
- `Ctrl+b, d` to detach session
- `tmux ls` to list sessions
- friendly guide: https://www.hamvocke.com/blog/a-quick-and-easy-guide-to-tmux/
- docs: https://github.com/tmux/tmux/wiki/Getting-Started

Reply:
Now added